service/task_manager_module.cc

```diff
@@ -82,6 +82,7 @@ static bool tablet_id_provided(const locator::tablet_task_type& task_type) {
 future<std::optional<tasks::virtual_task_hint>> tablet_virtual_task::contains(tasks::task_id task_id) const {
     co_await _ss._migration_manager.local().get_group0_barrier().trigger();
 
+    auto tmptr = _ss.get_token_metadata_ptr();
     auto tables = get_table_ids();
     for (auto table : tables) {
         auto& tmap = _ss.get_token_metadata().tablets().get_tablet_map(table);
```

Comment:
We should use the `tmptr` we already have, instead of re-requesting the token metadata.
I know this is of no consequence, but better to be consistent.